src/schema/sources.ts

```diff
@@ -1404,22 +1450,30 @@ export const resolvers: IResolvers<any, Context> = {
     publicSourceMemberships: async (
       _,
       { userId, ...args }: { userId: string } & ConnectionArguments,
-      ctx,
+      ctx: Context,
       info,
     ): Promise<Connection<GQLSourceMember>> => {
       return paginateSourceMembers(
         (queryBuilder, alias) => {
+          queryBuilder = queryBuilder.andWhere(`${alias}."userId" = :user`, {
+            user: userId,
+          });
```

Comment:
Not really related to types
```suggestion
          queryBuilder = queryBuilder.andWhere(`${alias}."userId" = :userId`, {
            userId: userId,
          });
```

Reply:
yeah, ill update